packages/vuetify/src/styles/settings/_utilities.scss

```diff
@@ -478,6 +478,32 @@ $utilities: map-deep-merge(
       property: position,
       class: position,
       values: static relative fixed absolute sticky
+    ),
+    // Sizing
+    "height": (
+      property: height,
+      class: h,
+      values: (
+        auto: auto,
+        screen: 100vh,
+        0: 0,
```

Comment:
wouldn't it be better if it matches the grid settings? so 0, 1/12, 2/12, etc 
probably 1/12 would be used frequently but 1/3 is definitely a common size

Reply:
what would the classes look like?

Reply:
Hmm maybe just 2 more - `w-33` and `w-66`?
Or actually i just looked at boostrap, they have only 25s, but they have a variable for this so if user could define other sizes with a variable i default 25s are ok. 
Btw they have also `mw-100` and `mh-100` classes